arithmetization/src/main/java/net/consensys/linea/zktracer/module/modexpdata/ModexpDataOperation.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright ConsenSys Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package net.consensys.linea.zktracer.module.modexpdata;
+
+import static net.consensys.linea.zktracer.types.Conversions.*;
+import static net.consensys.linea.zktracer.types.Conversions.bigIntegerToBytes;
+import static net.consensys.linea.zktracer.types.Utils.leftPadTo;
+import static net.consensys.linea.zktracer.types.Utils.rightPadTo;
+
+import java.math.BigInteger;
+
+import com.google.common.base.Preconditions;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.Accessors;
+import net.consensys.linea.zktracer.container.ModuleOperation;
+import net.consensys.linea.zktracer.types.UnsignedByte;
+import org.apache.tuweni.bytes.Bytes;
+
+@RequiredArgsConstructor
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = false)
+@Accessors(fluent = true)
+public class ModexpDataOperation extends ModuleOperation {
+  private static final int BEMR_LINE_COUNT = 32 * 4;
+  private static final int COMPONENT_SIZE = 512;
+
+  private final int hubStamp;
+
+  @EqualsAndHashCode.Include private final Bytes base;
+  @EqualsAndHashCode.Include private final Bytes exp;
+  @EqualsAndHashCode.Include private final Bytes mod;
+  @EqualsAndHashCode.Include private Bytes result;
```

Comment:
No need to include `result` in the hash; if `base`/`exp`/`mod` are already all equals, `result` will be the same as well.